common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -310,17 +310,27 @@ status_function() {
     if  pgrep -l beerocks_cont ; then
         echo "executing operational test using bml"
         if execute_beerocks_command "$bml_cmd" ; then
+            OUTPUT=$output
             # Expecting
             # > OK Main radio agent operational
             # > OK wlan0 radio agent operational
             # > OK wlan2 radio agent operational
-            OK_Count=$(echo "$output" | grep -c -e "OK.*operational")
+            OK_Count=$(echo "$OUTPUT" | grep -c -e "OK.*operational")
             if [ "$OK_Count" -eq 3 ]; then
                 success "operational test success!"
+                OUTPUT=$(echo "$OUTPUT" | grep -oP '(FAIL|OK).*?operational')
+                success "$OUTPUT"
                 exit 0
             else
                 err "operational test failed!"
-                err "$output"
+                OUTPUT=$(echo "$OUTPUT" | grep -oP '(FAIL| OK).*?operational' | sed -e "s/ OK/OK/")
```

Comment:
`-P` is not supported on all systems
```
grep: unrecognized option: P
BusyBox v1.30.1 () multi-call binary.
Usage: grep [-HhnlLoqvsriwFE] [-m N] [-A/B/C N] PATTERN/-e PATTERN.../-f FILE [FILE]...
Search for PATTERN in FILEs (or stdin)
	-H	Add 'filename:' prefix
	-h	Do not add 'filename:' prefix
	-n	Add 'line_no:' prefix
	-l	Show only names of files that match
	-L	Show only names of files that don't match
	-c	Show only count of matching lines
	-o	Show only the matching part of line
	-q	Quiet. Return 0 if PATTERN is found, 1 otherwise
	-v	Select non-matching lines
	-s	Suppress open and read errors
	-r	Recurse
	-i	Ignore case
	-w	Match whole words only
	-x	Match whole lines only
	-F	PATTERN is a literal (not regexp)
	-E	PATTERN is an extended regexp
	-m N	Match up to N times per file
	-A N	Print N lines of trailing context
	-B N	Print N lines of leading context
	-C N	Same as '-A N -B N'
	-e PTRN	Pattern to match
	-f FILE	Read pattern from file
```

Reply:
fixed, using -E instead